lib-py/casmutils/xtal/structure.py

```diff
@@ -0,0 +1,194 @@
+from . import _xtal
+from .lattice import Lattice, MutableLattice
+from .site import Site, MutableSite
+
+class _Structure():
+
+    """Base class for both mutable and immutable
```

Comment:
Explain a bit more what a structure is. At least mention that it's a lattice and basis.